index.js

```diff
@@ -243,6 +257,24 @@ exports.writeIcon = function (dir, cb) {
   })
 }
 
+exports.writeStore = function (dir, cb) {
+  var filename = path.join(dir, 'store.js')
+  var file = dedent`
+    module.exports = store
+
+    function store (state, emitter) {
+      state.totalClicks = 0
+
+      emitter.on('ADD_CLICK', function (count) {
```

Comment:
Perhaps we should be like super well-behaved, and wait for `'DOMContentLoaded'`? :grin: Doesn't matter too much here, but would improve load times slightly, and establish a pattern for people to rely on.

Also, most of our userland events (e.g. `choo-devtools`, `choo-tts`, `choo-serviceworker`) are written as `'<store-name>:<event-name>'`; perhaps keeping that pattern might guide people a little — so this would become `'clicks:add'`. Does that make sense?

Reply:
> wait for 'DOMContentLoaded'
That's a good practice. I will add that in. 

> this would become 'clicks:add'

Makes perfect sense. Can do. :+1: